docs/admin-guide/load-config.md

```diff
@@ -148,6 +115,9 @@ By giving a path to each individual vcf2cytosure-file these are made available
 for download on the case page. Such SV files can be visualized using standard arrayCGH
```

Comment:
This text here could be added to expand the field vcf2cytosure in the explanation above, and this part  could be removed

Reply:
Done!